packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -714,6 +714,38 @@ contract InvestmentPool is IInitializableInvestmentPool, SuperAppBase, Context,
         return subt / getPercentageDivider();
     }
 
+    /**
+     * @notice Function calculates the amount that is used by the creator
+     */
+    function getFundsUsed() public view returns (uint256) {
+        uint256 milestoneId = getCurrentMilestoneId();
+        uint256 currentState = getProjectStateByteValue();
+
+        if (
+            currentState == getCanceledProjectStateValue() ||
```

Comment:
I have a question here - these separate states - are they denoted with powers of 2?
If so - maybe we can make it a bit more elegant?